src/utils/googleAnalytics/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
```

Comment:
Can we add this to bento-tools in the future? Passing Tracking id from each data commons as a prop

Reply:
Yes. It was planned to add later to Bento-tools as a separate feature.